static/scripts/grml-config.sh

```diff
@@ -5,21 +5,26 @@ set -e
 # IMPORTANT: please note that you might override existing
 # configuration files in the current working directory!
 if command -v wget >/dev/null 2>&1 ; then
-  wget -O .screenrc     https://git.grml.org/f/grml-etc-core/etc/grml/screenrc_generic
-  wget -O .tmux.conf    https://git.grml.org/f/grml-etc-core/etc/tmux.conf
-  wget -O .vimrc        https://git.grml.org/f/grml-etc-core/etc/vim/vimrc
-  wget -O .zshrc        https://git.grml.org/f/grml-etc-core/etc/zsh/zshrc
-  wget -O .zshenv       https://git.grml.org/f/grml-etc-core/etc/zsh/zshenv
```

Comment:
Please include in your commit message why you removed `.zshenv` (or was it removed accidentally?).